mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -849,6 +849,53 @@ func TestAccContainerNodePool_shieldedInstanceConfig(t *testing.T) {
 	})
 }
 
+<% unless version.nil? || version == 'ga' -%>
```

Comment:
```suggestion
<% unless version == 'ga' -%>
```

Beta guards don't need to check for `nil` anymore, version is always filled in (but we definitely have not backfilled all the code)

Reply:
Thank you, I was confused about why in some places some of those where still checking for nil and just went with the style used in parts of tests.